docs/usage.md

```diff
@@ -48,9 +13,11 @@ NXF_OPTS='-Xms1g -Xmx4g'
 ```
 
 ## Running the pipeline
+
 The typical command for running the pipeline is as follows:
+
 ```bash
-nextflow run nf-core/epitopeprediction --reads '*_R{1,2}.fastq.gz' -profile standard,docker
+nextflow run nf-core/epitopeprediction --somatic_mutations '*.vcf.gz' -profile docker
```

Comment:
```suggestion
nextflow run nf-core/epitopeprediction --somatic_mutations "*.vcf.gz" -profile docker
```

Reply:
No tool specified here?

Reply:
Yeah, its the default - I made that now clear :-)